packages/@okta/vuepress-site/conductor.yml

```diff
@@ -2381,3 +2381,29 @@ redirects:
     to: /docs/guides/create-an-api-token/main/
   - from: /docs/guides/create-an-api-token/next-steps/index.html
     to: /docs/guides/create-an-api-token/main/
+  - from: /docs/guides/sign-your-own-saml-csr/overview/
+    to: /docs/guides/sign-your-own-saml-csr/main/
```

Comment:
In the past, we also had to include:
- from: /docs/guides/sign-your-own-saml-csr/overview/index.html
    to: /docs/guides/sign-your-own-saml-csr/main/

So, I've still been including the /index.html as well. If we don't still need to include those (not sure what would change to make that not necessary), then ignore this. But, if we do - you need one of those entries for each, also.